Fixed #24638 -- Added support for SQL comments (#18258)
django/django: docs/releases/5.2.txt

Models
~~~~~~

> * The new :meth:`.QuerySet.comment` method allows for injecting comments into
>   the generated SQL statements. This can be useful for correlating troublesome
>   queries to the application code.

Reviewer: Try to avoid judgmental words like “troublesome”, in favour of specific terms like “slow”. It’s more precise and translates better.

```suggestion
* The new :meth:`.QuerySet.comment` method allows for adding comments into
  the generated SQL. This can be useful for correlating slow queries with
  application code.
```
